utilities/pipelines/resourcePublish/Get-ModulesMissingFromPrivateBicepRegistry.ps1

```diff
@@ -71,7 +71,7 @@ function Get-ModulesMissingFromPrivateBicepRegistry {
         # Get all children
         $availableModuleTemplatePaths = (Get-ChildItem -Path (Split-Path $TemplateFilePath) -Recurse -Include @('deploy.bicep', 'deploy.json')).FullName
 
-        if (-not (Get-AzContainerRegistry -Name $BicepRegistryName -ResourceGroupName $BicepRegistryRgName)) {
+        if (-not (Get-AzContainerRegistry -Name $BicepRegistryName -ResourceGroupName $BicepRegistryRgName -ErrorAction 'SilentlyContinue')) {
```

Comment:
Does the fix allow the creation of the rg as well if not existing?

Reply:
The `Get-Modules(...)` functions don't create anything. They just check if the resouce is existing at all - and if not consider all modules worth publishing. It's only the following `Publish` functions that create the resources.

Reply:
I meant the general fix, I'm not referring to this file or line in particular. That's just to create a comment that can be easily resolved.

I understand this PR validates that a missing ACR does not cause the publishing to fail. My question is if the same stands for the RG hosting the specified ACR. So if neither the ACR nor the ACR RG exist, does the publishing create both?

Reply:
Yes it does. In the corresponding files for the registry and templates specs, the resources are fetched and, if not existing, created

Reply:
Woop woop